src/views/Graph.vue

```diff
@@ -91,7 +91,8 @@ const config = {
       css: {
         'background-image': function(node) {
           let path = node.data('icon')
-          path === undefined || path === '' ? path = require('@/assets/baseline-donut_large-24px.svg') : ''
+          // TODO: not sure if we can provide the path as a normal string, without importing the svg here?
```

Comment:
Oh, also added a TODO that I want to investigate later. I normally use `require` when importing things, but I need to check what's its return type... as I would expect the `path` variable here to be a string... just a note to self to clarify later (we may be able to use relative URL's, removing the need for require, which is easier to test)